### PR TITLE
AD operations.

### PR DESCRIPTION
**Pull request by TodereanLenuta**
Added sl's for:
- create computer accout,
- delete computer account,
- disable computer account,
- create user,
- delete user,
- enable user.

### CODE REVIEW

File: content/io/cloudslang/base/ldap/computers/create_computer_account.sl [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+#! @input execution_timeout: Time in milliseconds to wait for the connection to complete.
+#!                           Default value: 90000.
+#!                           Optional
+#!
+#! @output return_result: The return result of the operation.
+#! @output computer_distinguished_name: The distinguished name of the newly created computer account.
+#! @output return_code: The return code of the operation. 0 if the operation succeeded, -1 if the operation fails.
+#! @output exception: The exception message if the operation fails.
+#!
+#! @result SUCCESS: Generated description.
```
Comment (florinAndrei1991) [inline content/io/cloudslang/base/ldap/computers/create_computer_account.sl:46]: Missing description on both results.

File: content/io/cloudslang/base/ldap/computers/delete_computer_account.sl [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+#!                           Default value: 90000.
+#!                           Optional
+#!
+#! @output return_result: The return result of the operation.
+#! @output computer_distinguished_name: The distinguished name of the computer account that was deleted.
+#! @output return_code: The return code of the operation. 0 if the operation succeeded, -1 if the operation fails.
+#! @output exception: The exception message if the operation fails.
+#!
+#! @result SUCCESS: Generated description.
+#! @result FAILURE: Generated description.
```
Comment (florinAndrei1991) [inline content/io/cloudslang/base/ldap/computers/delete_computer_account.sl:44]: Missing description

File: content/io/cloudslang/base/ldap/computers/disable_computer_account.sl [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+#!                           Default value: 90000.
+#!                           Optional
+#!
+#! @output return_result: The return result of the operation.
+#! @output computer_distinguished_name: The distinguished name of the computer account that was disabled.
+#! @output return_code: The return code of the operation. 0 if the operation succeeded, -1 if the operation fails.
+#! @output exception: The exception message if the operation fails.
+#!
+#! @result SUCCESS: Generated description.
+#! @result FAILURE: Generated description.
```
Comment (florinAndrei1991) [inline content/io/cloudslang/base/ldap/computers/disable_computer_account.sl:39]: Missing description

File: content/io/cloudslang/base/ldap/users/create_user.sl [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+#!                           Optional
+#!
+#! @output return_result: A message with the common name of the newly created user in case of success or the error
+#!                        message in case of failure.
+#! @output user_distinguished_name: The distinguished name of the newly created user.
+#! @output return_code: The return code of the operation. 0 if the operation succeeded, -1 if the operation fails.
+#! @output exception: The exception message if the operation fails.
+#!
+#! @result SUCCESS: Generated description.
+#! @result FAILURE: Generated description.
```
Comment (florinAndrei1991) [inline content/io/cloudslang/base/ldap/users/create_user.sl:55]: Same as above

File: content/io/cloudslang/base/ldap/users/delete_user.sl [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+#!                           Optional
+#!
+#! @output return_result: A message with the common name of the deleted user in case of success or the error message in
+#!                        case of failure.
+#! @output user_distinguished_name: The distinguished name of the deleted created user.
+#! @output return_code: The return code of the operation. 0 if the operation succeeded, -1 if the operation fails.
+#! @output exception: The exception message if the operation fails.
+#!
+#! @result SUCCESS: Generated description.
+#! @result FAILURE: Generated description.
```
Comment (florinAndrei1991) [inline content/io/cloudslang/base/ldap/users/delete_user.sl:46]: Same

File: content/io/cloudslang/base/ldap/users/enable_user.sl [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+#!                           Optional
+#!
+#! @output return_result: A message with the common name of the enabled user in case of success or the error message in
+#!                        case of failure.
+#! @output user_distinguished_name: The distinguished name of the enabled user.
+#! @output return_code: The return code of the operation. 0 if the operation succeeded, -1 if the operation fails.
+#! @output exception: The exception message if the operation fails.
+#!
+#! @result SUCCESS: Generated description.
+#! @result FAILURE: Generated description.
```
Comment (florinAndrei1991) [inline content/io/cloudslang/base/ldap/users/enable_user.sl:46]: Here as well

File: content/io/cloudslang/base/ldap/computers/delete_computer_account.sl (hunk 2 above)
Comment (TodereanLenuta) [reply content/io/cloudslang/base/ldap/computers/delete_computer_account.sl]: Added the desc in the last commit.

**Review by florinAndrei1991 — APPROVED** [review]

**Review by IuliaMoldovan — APPROVED** [review]

**Pull request merged**